samples/shields/x_nucleo_iks01a3/sensorhub/CMakeLists.txt

```diff
@@ -7,8 +7,6 @@ cmake_minimum_required(VERSION 3.13.1)
 # This sample is specific to x_nucleo_iks01a3 shield. Enforce -DSHIELD option
 set(SHIELD x_nucleo_iks01a3_shub)
 
-# IKS01A3 board configured in sensorhub mode
-set(CONF_FILE shub.conf)
 
```

Comment:
Actually I had in the plan to do it as well. Someone suggested to me that prj.conf should have been the name used.